bin/agd

```diff
@@ -14,6 +14,11 @@
 # the SDK.
 set -ueo pipefail
 
+function fatal() {
+  echo $1 1>&2
```

Comment:
Quoting.
```suggestion
  echo "$1" 1>&2
```

Reply:
The `echo` command is happy to echo multiple arguments, though if you omit the double quotes it might normalize whitespace and interpret a leading `"-n "`, though I'd argue that the latter is a feature. Nevertheless I'll make the change because of my philosophy which departs from Emerson: "A foolish consistency is the hobgoblin of good programming."

Reply:
BTW, if you want the `fatal()` function to be variadic and treat arguments exactly as `echo` does, may I suggest:

```sh
  echo ${1+"$@"} 1>&2
```

Reply:
Done.